src/core/Form/Toggle/Toggle.baseStyles.tsx

```diff
@@ -13,9 +13,30 @@ export const baseStyles = withSuomifiTheme(
   ${element({ theme })}
   ${font({ theme })('bodyText')}
   background-color: ${theme.colors.whiteBase};
-  & > .fi-toggle_label {
-    cursor: pointer;
-  }
+    & + .fi-toggle--with-button {
```

Comment:
Are these changes a bug fix to something that was broken earlier?

Reply:
If you mean the `& > .fi-toggle_label {` it was removed as it has no effect after the structure change of input-version. 
The cursor is still correct, `pointer` so didn't see need to add separate rule to set it to `pointer` again.

Reply:
And this was tested using different browsers?

Reply:
On desktop tested with Safari and Chrome. And Safari on mobile.

_Additional note, VoiceOver will read the state change with Space and Enter on desktop Chrome but not with Safari._